### DISCUSSION TITLE
blog/reason-and-nature

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#15] [Announcements]
# blog/reason-and-nature

麦克道威尔《心灵与世界》一书最有价值的一部分就是第四讲“理性与自然”。在那里，他处理了许多重大的哲学问题，特别是理性的自然化问题：理性会是一种自然的东西吗？如果是，它属于什么意义上的自然的东西？此外，如果将理性自然化，这将如何协调其对经验思维的构想？这一系列的问题涉及到近代以来有关自然的观念的转变，以及“意义”这个特殊的东西所面临的存在论威胁，此外还涉及传统哲学的主要担忧，特别重要的是，涉及到了伦理事项的存在论地位问题。麦克道威尔对于这些问题都给予了极具启发性的思考。

https://rercel.com/blog/reason-and-nature

**Comment by DayuGuo** [comment]
昨天看完你的博客就沉沉的睡了，很难得的做了梦，我似乎已经几年没有梦的体验。
似乎是在一个房间里，有奇怪的生物试图冲进来。人形的，五颜六色带着线条的奇怪生物。
我一直顶着门不让他们进，但是没人帮我。不断大喊着，没有回应。
